defold-spine/commonsrc/vertices.cpp

```diff
@@ -100,6 +100,11 @@ uint32_t GenerateVertexData(dmArray<SpineVertex>& vertex_buffer, const spSkeleto
             EnsureArrayFitsNumber(scratch, 4*2);
             EnsureArrayFitsNumber(vertex_buffer, 6);
 
+            float colorR = tintR * regionAttachment->color.r;
+            float colorG = tintG * regionAttachment->color.g;
+            float colorB = tintB * regionAttachment->color.b;
+            float colorA = tintA * regionAttachment->color.a;
+
```

Comment:
Their documentation/examples doesn't cover this, but I think this is how it's supposed to work.
http://esotericsoftware.com/spine-c#Implementing-Rendering